src/main/java/org/apache/maven/plugins/jlink/JLinkMojo.java

```diff
@@ -654,7 +656,7 @@ private List<String> createJlinkArgs( Collection<String> pathsOfModules,
         if ( addOptions != null && !addOptions.isEmpty() )
         {
             jlinkArgs.add( "--add-options" );
-            jlinkArgs.add( String.format( "\"%s\"", String.join( " ", addOptions ) ) );
+            jlinkArgs.add( String.format( "%s", String.join( " ", addOptions ) ) );
```

Comment:
We can support that here. We would just need to iterate over `addOptions` instead, so -1 on this one.

Reply:
I think I did this because jlink can't handle it properly